src/views/product-downloads-view/server.ts

```diff
@@ -15,10 +16,22 @@ import {
 /**
  * Prepares static props for downloads views
  */
-async function generateStaticProps(
-	productData: ProductData,
-	pageContent: RawProductDownloadsViewContent
-): Promise<GetStaticPropsResult<ProductDownloadsViewStaticProps>> {
+async function generateStaticProps(productData: ProductData): Promise<{
+	props: ProductDownloadsViewStaticProps
+	revalidate: number
+}> {
+	/**
+	 * Note: could consider other content sources. For now, JSON.
+	 * Asana task: https://app.asana.com/0/1100423001970639/1201631159784193/f
+	 */
+	const jsonFilePath = path.join(
+		process.cwd(),
+		`src/content/${productData.slug}/install-landing.json`
+	)
+	const pageContent: RawProductDownloadsViewContent = JSON.parse(
+		fs.readFileSync(jsonFilePath, 'utf8')
+	)
```

Comment:
🗒️ This update was made to align with how we read from content files in other views. Also generating the file path from the product slug, as mentioned in previous comment.